rs/tests/boundary_nodes/rate_limit_canister_test.rs

```diff
@@ -0,0 +1,390 @@
+/* tag::catalog[]
+Title:: Setting rate-limits on the API boundary nodes via rate-limit canister
+
+Goal:: Verify that API boundary nodes dynamically fetch rate-limit configurations from the canister and enforce them for ingress messages.
```

Comment:
The goal talks about API boundary nodes but I don't see those being deployed in the setup function. What's the deal with that?

Reply:
As discussed offline, ATM this is rather a placeholder for the future comprehensive `system-test`. Currently we just install the canister and check that it works. Later we will extend the setup with API boundary nodes and enable `ic-boundary` service talk to the canister. Bear with us. 🐻

Reply:
Ok but could this be added to the description since it's confusing when reading the code without this context.

Reply:
[done](https://github.com/dfinity/ic/pull/1961/commits)